src/cyclebane/node_values.py

```diff
@@ -45,6 +48,17 @@ def from_array_like(values: Any, *, axis_zero: int = 0) -> ValueArray:
     @abstractmethod
     def try_from(obj: Any, *, axis_zero: int = 0) -> ValueArray | None: ...
 
+    def __eq__(self, other: object) -> bool:
+        if type(self) != type(other):
+            return False
```

Comment:
```suggestion
            return NotImplemented
```
To get the proper fallback.